framework/src/modules/dpos/transaction_assets/pom_transaction_asset.ts

```diff
@@ -155,7 +155,7 @@ export class PomTransactionAsset extends BaseAsset<PomTransactionAssetContext> {
 		}
 
 		if (
-			getPunishmentPeriod(
+			getUnlockDelayPeriod(
```

Comment:
Should we rename the function here too?
I think that PoM is different than unlock. Maybe it makes sense to keep the name here? I see that this is a bit conflicting as this function is used in both `unlock` and `pom`. 
With the way you made the changes, I have to say that keeping `getPunishmentPeriod` looks also good to me.